### PR TITLE
tree: Fix dist imports in tests

### PR DESCRIPTION
**Pull request by CraigMacomber**
## Description

Rename a test file missing its `.spec`, fix imports from `dist`, improve tree workspace settings to not recommend dist imports and relative imports form other packages.

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

### CODE REVIEW

File: packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts [hunk 1]
```diff
@@ -54,7 +54,6 @@ import { SchemaFactory, TreeConfiguration, toFlexConfig } from "../../../simple-
 // eslint-disable-next-line import/no-internal-modules
 import { toFlexSchema } from "../../../simple-tree/toFlexSchema.js";
 // eslint-disable-next-line import/no-internal-modules
```
Comment (daesunp) [inline packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts:56]: Is this eslint comment still used?
```suggestion
```

Comment (CraigMacomber) [reply packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts:56]: No, it should not be. Good catch!

File: packages/dds/tree/.vscode/Tree.code-workspace [hunk 2]
```diff
@@ -10,6 +10,10 @@
 		}
 	],
 	"settings": {
-		"search.followSymlinks": false
+		"search.followSymlinks": false,
+		"typescript.preferences.autoImportFileExcludePatterns": [
```
Comment (Abe27342) [inline packages/dds/tree/.vscode/Tree.code-workspace:14]: seems like a good thing to include in the root workspace as well (.vscode/settings.json). If you don't want to do that here, I can make a separate PR for it. let me know :)

Comment (CraigMacomber) [reply packages/dds/tree/.vscode/Tree.code-workspace:14]: I'm going to try and land this as is to unblock the perf tests pipeline. Moving this to the folder settings (no need for it to be in both), possible along with `"typescript.preferences.preferTypeOnlyAutoImports": true` seems like a good idea if they work well for all the packages.

Comment (Abe27342) [reply packages/dds/tree/.vscode/Tree.code-workspace:14]: sounds good to me

**Review by Abe27342 — APPROVED** [review]

**Pull request merged**